dobi.yaml

```diff
@@ -374,6 +386,7 @@ compose=integration-env:
     # rust images
     - analyzer-dispatcher
     - generic-subgraph-generator
+    - metric-forwarder # TODO should I???
```

Comment:
since we haven't figured out a way to loop this into integration yet, unsure if it's worth including here

Reply:
IMO definitely worth including it here, if only as a reminder that it exists. This list should stay in sync with the docker-compose.yml--e.g. every service in that file should be represented here.